subsys/bluetooth/Kconfig

```diff
@@ -109,6 +109,17 @@ config BT_MAX_CONN
 	  Maximum number of simultaneous Bluetooth connections
 	  supported.
 
+config BT_CONN_DYNAMIC_CALLBACKS
+	bool "Dynamic connection callbacks"
+	depends on BT_CONN
```

Comment:
There's a separate `if BT_CONN` section a little bit further down. I think it makes sense to move this there, and then you can remove the `depends on BT_CONN` since it becomes redundant.

Reply:
Sounds good, just a question. Is there any difference in using `depends on` vs a `if block`?